src/wallets/browser/NearWallet.ts

```diff
@@ -49,7 +56,7 @@ export default class NearWallet extends BrowserWallet implements INearWallet {
   }
 
   async getAccount(): Promise<any> {
-    if (!this.isConnected()) return null;
+    if (!(await this.isConnected())) return null;
```

Comment:
This is another case where the method isn't really async. [`this.wallet.isSignedIn()`](https://near.github.io/near-api-js/classes/walletaccount.walletconnection.html#issignedin) actually returns a boolean. We need to remove the `async` keyword on methods that are synchronous (e.g. `isConnected`, `signIn`, `disconnect`, etc).

Reply:
The reason we are using async in some of the methods NearWallet.ts even though we might not need it is because we are extending BrowserWallet which implements BaseWallet, the async in those methods I think is needed in other wallets like, Ledger or Sender Wallet.